src/models/multiBarChart.js

```diff
@@ -334,12 +334,14 @@ nv.models.multiBarChart = function() {
 
     multibar.dispatch.on('elementMouseover.tooltip', function(evt) {
         evt.value = chart.x()(evt.data);
-        evt['series'] = {
-            key: evt.data.key,
-            value: chart.y()(evt.data),
-            color: evt.color
-        };
-        tooltip.data(evt).hidden(false);
+        if (!chart.stacked() || (evt.data.size > 0 || evt.data.size == undefined)) {
```

Comment:
I don't see how though... I don't see where it's set anywhere in multiBar.js, and that's what passes the data on the "mouseover" d3 event...   and I can't make it have a size attribute using the multiBar.html or multiBar2.html examples in the examples folder...